src/lib.sw

```diff
@@ -9,6 +9,7 @@ dep contract_id;
 dep context;
 dep address;
 dep block;
+dep token;
```

Comment:
Hmm. Can you sort these?

Reply:
these have to be in a specific order to work.
Specifically, a dependency must be listed before the lib which depends on it.